pkgs/development/libraries/bullet/default.nix

```
@@ -40,7 +40,10 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
-  NIX_CFLAGS_COMPILE = stdenv.lib.optionalString stdenv.cc.isClang "-Wno-error=argument-outside-range";
+  NIX_CFLAGS_COMPILE = stdenv.lib.optionals stdenv.cc.isClang [
```

```suggestion
  NIX_CFLAGS_COMPILE = stdenv.lib.optionalString stdenv.cc.isClang
    "-Wno-error=argument-outside-range -Wno-error=c++11-narrowing"
  ];
```

Unlike eg. buildInputs NIX_* attributes are only used as environment variables which should be strings. While lists work sometimes it could lead to unexpected results.